src/App.css

```diff
@@ -35,27 +36,41 @@ a:hover {
 h1 {
   padding-bottom: 20px;
   color: rgb(255, 255, 255);
-  font-size: 1.3em;
+  font-size: 1.4em;
   text-align: center;
 }
 
 h2 {
-  padding: 30px;
   color: #5e5e5e;
-  font-size: 1.0em;
-  
+  font-size: 1.2em;
 }
 
 h3 {
   color: #2d2d2d;
-  font-size: 1.4em;
+  font-size: 1.3em;
   margin-bottom: 15px;
 }
 
 h4 {
   color: #727272;
-  font-size: 1.0em;
+  font-size: 1.3em;
   line-height: 160%;
 }
 
+.logo {
+  width: 190px;
```

Comment:
Just wondering: why is `px` (absolute measurment) used for sizes (width, padding, top, etc.) and `em` (relative measurment) for fonts? 🤔

Reply:
I have been a bit inconsistent there 😅  What should I use?

Reply:
I would say: it depends. Every team / project has it's own rules. Consistency in general is just a good rule to follow.
Maybe on a project the team agrees that they would use `em` for fonts (for a given reason).

So in this case it's up to you (as it's your project).

As I wrote: I was just wondering if there's some reasoning behind this.